AsyncDisplayKit/ASTableView.h

```diff
@@ -133,6 +134,14 @@
  */
 - (NSArray *)visibleNodes;
 
+/**
+ * YES to automatically adjust the contentOffset when cells are inserted or deleted "before"
+ * visible cells, maintaining the users' visible scroll position. Very useful when supporting virtualization.
```

Comment:
Expand comment to describe "virtualization", or let's leave it out.  "Data source virtualization" is probably a good way to refer to it, but even with the extra words, I think more than half of developers will not really know what this means.